azure-pipelines.yml

```diff
@@ -40,6 +40,15 @@ jobs:
       Catalina-Xcode-11.4:
         IMAGE_POOL: 'macOS-10.15'
         XCODE_VERSION: '11.4'
+      Catalina-Xcode-11.6:
+        IMAGE_POOL: 'macOS-10.15'
+        XCODE_VERSION: '11.6'
+      Catalina-Xcode-11.7:
```

Comment:
My advice is to only test on:

* Catalina Xcode 11.7
* Big Sur Xcode 11.7
* Catalina Xcode 12.0 beta
* Big Sure Xcode 12.0 beta

All other Xcode versions are irrelevant now.

Reply:
Thanks for your comment
AFAIK, ADO don't have Big Sur image right now. What do you think about updating test matrix right after they add this image?
For now I can left only the following test matrix if the previous Xcode versions are irrelevant:
- Catalina Xcode 11.7
- Catalina Xcode 12 beta

Reply:
^ those 2 are enough for this time of year.

Reply:
@jmoody Done 🙂

Reply:
Great. :)